shared/params/config.go

```diff
@@ -234,9 +234,9 @@ func MainnetConfig() *BeaconChainConfig {
 func DemoBeaconConfig() *BeaconChainConfig {
 	demoConfig := MinimalSpecConfig()
 	demoConfig.MinDepositAmount = 100
-	demoConfig.MaxEffectiveBalance = 3.2 * 1e9
-	demoConfig.EjectionBalance = 1.6 * 1e9
-	demoConfig.EffectiveBalanceIncrement = 0.1 * 1e9
+	demoConfig.MaxEffectiveBalance = 3.2 * 1e6
```

Comment:
???

Reply:
oh , sorry . It was a config change for local testing. Reverting this